### DISCUSSION TITLE
Important Update

### DISCUSSION THREAD

**Discussion by peterantypas** [peterantypas/maiana#61] [General]
I have a few announcements:

- My main project just got cancelled for a variety of reasons
- I will be leaving my current employer at some point in the very near future (but not immediately)
- I now have time to make MAIANA happen, despite the state of the supply chain

If you want to be kept up-to-date on the status of the kit, please sign up here:

http://eepurl.com/hxHP-5

Talk soon,
Peter

**Comment by jrcardwell** [comment]
Sorry to hear that , Peter.  Been there and I know it’s disappointing, not to mention disrupting.  I’m sure the next gig will be as or more rewarding.  Then there’s Mariana... :)

Take care,

Jim

**Comment by muebau** [comment]
Wow thats hard. All the best and good luck for you.

**Comment by Max194703** [comment]
Ciao Peter, I'm sorry and I wish you all the best, let me know and ... good wind, always!

**Comment by rfe3** [comment]
Any update on the MAIANA Kits? I'd really love to getz a hold of one of these....

**Comment by peterantypas** [comment]
If you're looking for a kit, they are in fact available. I have created a dedicated email address for this, so please contact me at:

maiana.kits@gmail.com

Thank you.

**Comment by peterantypas** [comment]
Well, yes, they exist. Sorry, I haven't been sending regular updates on the
mailing list lately (are you subscribed?)

What kind of interface are you looking for? There is USB, NMEA0183+USB and
NMEA2000+USB.

**Reply by rfe3** [reply]
Cool, yes I'm on the mailing list but haven't heard anything so I thought I'd ask...I'm not sue about the interface, I'm just starting to put my system together and do some testing with it. I have a Rpi 4 and a Montisser Hat that I'll use as a reciever but a transponder is what I really want...Which interface would you recommend?

**Reply by gah67** [reply]
Maiana does everything the Moitessier Hat does and more. Unlike the hat, Maiana has the antenna built-in and, as you know, it is also a transponder.
Maiana also has 'GPS' and dual-channel receiver etc and, without the need for coax and other connectors, it should actually perform better as well.

I am using the NMEA0183+USB version and I will be connecting it to a rPi 4 very soon (using the NMEA connection via a cheap USB RS-422 converter)

(it's already working very well and I have already used the USB connection straight into my laptop running OpenCPN)
